docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -52,13 +56,99 @@ Capacity metrics are currently not supported via Diagnostic Settings.
+
+* **Activity Logs Location Extraction FER**
+
+   ```sql
+   Rule Name: AzureLocationExtractionFER
+   Applied at: Ingest Time
+   Scope (Specific Data): tenant_name=*
+   ```
+
+   ```sql title="Parse Expression"
+   json "location", "properties.resourceLocation", "properties.region" as location, resourceLocation, service_region nodrop
+   | replace(toLowerCase(resourceLocation), " ", "") as resourceLocation
+   | if (!isBlank(resourceLocation), resourceLocation, location) as location
+   | if (!isBlank(service_region), service_region, location) as location 
+   | if (isBlank(location), "global", location) as location
+   | fields location
+   ```
+
+* **Resource ID Extraction FER**
+
+   ```sql
+   Rule Name: AzureResourceIdExtractionFER
+   Applied at: Ingest Time
+   Scope (Specific Data): tenant_name=*
+   ```
+
+   ```sql title="Parse Expression"
+   json "resourceId"
+   | toUpperCase(resourceId) as resourceId
+   | parse regex field=resourceId "/SUBSCRIPTIONS/(?<subscription_id>[^/]+)" nodrop
+   | parse field=resourceId "/RESOURCEGROUPS/*/" as resource_group nodrop
+   | parse regex field=resourceId "/PROVIDERS/(?<provider_name>[^/]+)" nodrop
+   | parse regex field=resourceId "/PROVIDERS/[^/]+(?:/LOCATIONS/[^/]+)?/(?<resource_type>[^/]+)/(?<resource_name>.+)" nodrop
+   | parse regex field=resource_name "(?<parent_resource_name>[^/]+)(?:/PROVIDERS/[^/]+)?/(?<service_type>[^/]+)/?(?<child_service_name>.+)" nodrop
+   | if (isBlank(parent_resource_name), resource_name, parent_resource_name) as resource_name
+   | fields subscription_id, location, provider_name, resource_group, resource_type, resource_name, service_type
+   ```
+### Configure metric rules
+* **Azure Observability Metadata Extraction Metric Rule Service Level**
+```sql
+Rule Name: AzureObservabilityMetadataExtractionMetricRuleServiceLevel
+```
+
+```sql title="Metric match expression"
+resourceId=/SUBSCRIPTIONS/*/RESOURCEGROUPS/*/PROVIDERS/*/*/*/*/* tenant_name=*
+```
+| Fields extracted | metric rule    |
+|------------------|----------------|
+| subscription_id  | $resourceId._1 |
+| resource_group   | $resourceId._2 |
+| provider_name    | $resourceId._3 |
+| resource_type    | $resourceId._4 |
+| resource_name    | $resourceId._5 |
+| service_type     | $resourceId._6 |
+
+* **Azure Observability Metadata Extraction Storage Account Level**
+```sql
+Rule Name: AzureObservabilityMetadataExtractionStorageAccountLevel
+```
+
+```sql title="Metric match expression"
+resourceId=*/SUBSCRIPTIONS/*/RESOURCEGROUPS/*/PROVIDERS/*/*/* tenant_name=* accountresourceid=*
+```
+| Fields extracted | metric rule    |
+|------------------|----------------|
+| subscription_id  | $resourceId._1 |
+| resource_group   | $resourceId._2 |
+| provider_name    | $resourceId._3 |
+| resource_type    | $resourceId._4 |
+| resource_name    | $resourceId._5 |
 
 ### Configure metrics collection
```

Comment:
Before metrics collection we should add a instruction of creating hosted collector and tagging tenant_name field along with the screenshot.